visidata/loaders/toml.py

```diff
@@ -0,0 +1,61 @@
+from visidata import (
+    ColumnItem,
+    PythonSheet,
+    VisiData,
+    asyncthread,
+    deduceType,
+    vd,
+)
+
+
+@VisiData.api
+def open_toml(vd, p):
+    return TomlSheet(p.name, source=p)
+
+
+class TomlSheet(PythonSheet):
+    """A Sheet representing the top level of a loaded TOML file.
+
+    This is an intentionally minimal loader with cues taken from
+    VisiData built-in JSON and Python object sheet types.
+    """
+    help = '''# Toml Sheet
+This sheet represents the top level of {sheet.source.name}.{sheet.source.ext}.
+
+Each cell within this sheet can contain dictionaries (representing TOML key:value pairs), lists (representing TOML arrays), or scalars.
+
+Some helpful commands when working with cells of lists and dictionaries:
+
+- `zEnter` on a cell with a dictionary will "dive" into the current cell, creating a seperate sheet with keys as **Columns** and values as **Rows**.
+- `(` (`expand-col`) on a column with lists or dictionaries will "expand" the structures in the cells into new columns within the current sheet.
```

Comment:
Good call, thanks for adding this! You're right, it's hard to avoid jargon when explaining dives/expands. It's also a bit awkward because the row/column layout you get from `zEnter` varies depending on whether you're diving into a list, dict, list of dicts...

It might be cheating to avoid jargon by also being less specific, but maybe we can highlight how both commands zoom into nested data. And the difference is whether that'll happen here or in a new sheet, like...

```suggestion
- `(` (`expand-col`) on a column with lists or dictionaries will "expand" the structures in the cells into new columns within the current sheet.
- `zEnter` on a cell with lists or dictionaries will "dive" into the current cell, expanding its structures into rows and columns in a seperate sheet.
```

Reply:
Great! Love it!